zcash_client_sqlite/src/testing/pool.rs

```diff
@@ -951,7 +961,7 @@ pub(crate) fn change_note_spends_succeed<T: ShieldedPoolTester>() {
     let dfvk = T::test_account_fvk(&st);
 
     // Add funds to the wallet in a single note owned by the internal spending key
-    let value = NonNegativeAmount::const_from_u64(60000);
+    let value = NonNegativeAmount::const_from_u64(70000);
```

Comment:
Why is this amount increased?

Reply:
This change is no longer necessary. I think it was because we were previously calculating the fee incorrectly.